biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -136,6 +136,7 @@
 	String				EXECUTABLE									= "-executable";
 	String				EXPORT										= "-export";
 	String				EXPORTTYPE									= "-exporttype";
+	String				EXPORT_APIGUARDIAN							= "-export-apiguardian";
```

Comment:
Need to add to `BUNDLE_SPECIFIC_HEADERS` and `options` in this class.

Need to add to `Syntax` class.

Reply:
done!